Dockerfile

```
@@ -41,12 +41,11 @@ COPY schema ./schema
 COPY scripts ./scripts
 COPY tools ./tools
 COPY vast ./vast
-COPY .clang-format .cmake-format LICENSE LICENSE.3rdparty README.md BANNER CHANGELOG.md CMakeLists.txt configure vast.yaml.example ./
-RUN ./configure \
-    --prefix=$PREFIX \
-    --build-type=$BUILD_TYPE \
-    --log-level=INFO \
-    --generator=Ninja
+COPY .clang-format .cmake-format LICENSE LICENSE.3rdparty README.md BANNER CHANGELOG.md CMakeLists.txt vast.yaml.example ./
+RUN cmake -B build -G Ninja \
+    -DCMAKE_INSTALL_PREFIX="$PREFIX" \
+    -DCMAKE_BUILD_TYPE="$BUILD_TYPE" \
+    -DCMAKE_VAST_LOG_LEVEL=INFO
```

The variable for the log level has no `CMAKE_` prefix, but I don't think we should use it anyways, but rather rely on the defaults for the build type.

```suggestion
    -DCMAKE_BUILD_TYPE:STRING="$BUILD_TYPE"
```

---

I just changed the variable name now to keep the end result the same as before.